jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/question.vm

```diff
@@ -107,6 +107,10 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
       #if(!${currentUser.anonymous} && ${post.userCreated.id} == ${currentUser.id})
         #set($own = true)
       #end
+      #set($labelAnswer = "label.answer")
+      #if($own)
```

Comment:
This condition is redundant.  We can determene which label to use without this flag.